src/DssSpell.t.sol

```diff
@@ -1415,4 +1416,18 @@ contract DssSpellTest is DSTest, DSMath {
         checkAuth(true);
     }
 
+    function test_interim_budget() public {
+        address multisig = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
+        uint256 prevSin = vat.sin(address(vow));
+        uint256 prevDai = dai.balanceOf(multisig);
+
```

Comment:
Can we test that the allowance is `0` prior to and after the spell as well. We're setting it to `1` in there but not checking if there's already an allowance.